Tests/NostrSDKTests/EventDecodingTests.swift

```diff
@@ -532,7 +532,7 @@ final class EventDecodingTests: XCTestCase, FixtureLoading {
         
         XCTAssertTrue(event.noteIds.contains("be8567dc210986fe5dc0ad04e02a8850b3b86e1d30c0ab674d102f0eefa68921"))
         XCTAssertTrue(event.hashtags.contains("up"))
-        XCTAssertTrue(event.links.contains(URL(string: "https://nostr.com/")!))
+        XCTAssertTrue(event.references.contains(URL(string: "https://nostr.com/")!))
```

Comment:
Should we keep this as `links` as that's the attribute that we are exposing that's closer to the language of the NIP?

Whatever we decide to do here in tests, I think we should be consistent with testing the other attribute `articlesCoordinates` / `eventCoordinates` on line 540.

Reply:
Yep, it was just an oversight. Thanks!